src/main/java/michael/Task.java

```diff
@@ -0,0 +1,35 @@
+package michael;
+
+public class Task {
+    private final String taskName;
+    private boolean done;
```

Comment:
Would it be better to name the variable with another name that would immediately signal it as a boolean variable?
```suggestion
    private boolean isDone;
```